src/Aspire.Dashboard/Components/Controls/PropertyGrid.razor

```diff
@@ -7,10 +7,10 @@
                 Style="width:100%"
                 GenerateHeader="GenerateHeaderOption.Sticky"
                 GridTemplateColumns="@GridTemplateColumns">
-    <TemplateColumn Title="@(NameColumnTitle ?? Loc[ControlsStrings.PropertyGridNameColumnTitle])" Class="nameColumn" SortBy="@NameSort" Sortable="@IsNameSortable">
+    <TemplateColumn Title="@(NameColumnTitle ?? Loc[ControlsStrings.NameColumnHeader])" Class="nameColumn" SortBy="@NameSort" Sortable="@IsNameSortable">
         <GridValue Value="@NameColumnValue(context)" HighlightText="@HighlightText" />
     </TemplateColumn>
-    <TemplateColumn Title="@(ValueColumnTitle ?? Loc[ControlsStrings.PropertyGridValueColumnTitle])" Class="valueColumn" SortBy="@ValueSort" Sortable="@IsValueSortable">
+    <TemplateColumn Title="@(ValueColumnTitle ?? Loc[ControlsStrings.PropertyGridValueColumnHeader])" Class="valueColumn" SortBy="@ValueSort" Sortable="@IsValueSortable">
```

Comment:
Why the generic `Name` but specific `PropertyGridValue`? I would expect `Value` to be just as generic.

Looks like you initially started with specific names for everything but later merged some?

Reply:
Yeah, column names used by multiple components were moved to `ControlsStrings`. It was intentional :)